### PR TITLE
Syangja Added

### PR DESCRIPTION
**Pull request by ori-on1**
## What kind of change does this PR introduce?

- [x] Bug Fix
- [x] Documentation change
- [x] Added a new city
- [x] Other(s)

### CODE REVIEW

File: data/borderlines.json [hunk 1]
```diff
@@ -127,6 +127,42 @@
...
+        },
+        {
+          "name": "Sworek Maidan",
+          "latitude": 28.0248,
+          "longitude": 83.7354
+        },
+        {
+          "name": "Bahunthan Hill",
+          "latitude": 27.9896,
+          "longitude": 83.7456
```
Comment (thenishantsapkota) [inline data/borderlines.json:162]: Can we add a bit more landmarks please?
The most significant ones

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/borderlines.json (hunk 1 above)
Comment (ori-on1) [reply data/borderlines.json:177]: Please review, more landmarks are added !

File: data/borderlines.json [hunk 2]
```diff
@@ -127,6 +127,57 @@
...
+      "currency": {
+        "name": "Nepalese Rupee",
+        "code": "NPR"
+      },
+      "landArea": {
+        "value": 1164,
+        "unit": "km²"
+      },
+      "elevation": {
+        "value": 1261,
```
Comment (thenishantsapkota) [inline data/borderlines.json:145]: I think the elevation value isn't correct for this?
What is the source of this data?

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/borderlines.json (hunk 2 above)
Comment (ori-on1) [reply data/borderlines.json]: corrected elevation, please review again. The source of the data is topographic-map.com

**Pull request closed without merging**